### PR TITLE
Laravel 8 based

### PR DESCRIPTION
**Pull request by mrmacgeek**
Pull request as anounced of discord

### CODE REVIEW

File: app/Helpers/XenforoHelper.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+namespace App\CustHelpers;
+
+
+use Illuminate\Support\Facades\Http;
+use RestCord\DiscordClient;
+
+class XenforoHelper
+{
+
+    const BASE_URL = 'https://pokemon3d.net/forum/api';
```
Comment (dsbilling) [inline app/Helpers/XenforoHelper.php]: This should be a environment variable

File: app/Helpers/GitHubHelper.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+class GitHubHelper
+{
+    private $githubData;
+
+    public function __construct()
+    {
+        $this->githubData = Cache::get("current_version",
+            function () {
+                $ApiUrl = "https://api.github.com/repos/P3D-Legacy/P3D-Legacy/releases/latest";
```
Comment (dsbilling) [inline app/Helpers/GitHubHelper.php]: This should also be a environment variable

**Review by dsbilling — CHANGES_REQUESTED** [review]

File: app/Helpers/GitHubHelper.php (hunk 2 above)
Comment (mrmacgeek) [reply app/Helpers/GitHubHelper.php]: addressed in commit #62b5ad5

File: app/Helpers/XenforoHelper.php (hunk 1 above)
Comment (mrmacgeek) [reply app/Helpers/XenforoHelper.php]: addressed in commit #62b5ad5

File: .env.example [hunk 3]
```diff
@@ -52,4 +52,7 @@ DISCORD_TOKEN=
 DISCORD_SERVER_ID=
 DISCORD_INVITE_URL="https://discordapp.com/invite/EUhwdrq"
 
-XENFORO_API_KEY=
+GITHUB_BASE_URL="https://api.github.com/repos/P3D-Legacy/P3D-Legacy"
+
+XENFORO_BASE_URL="https://pokemon3d.net/forum/api"
+XENFORO_API_KEY=enuGSgamqgPxgLH_UymTp04u74KVUqEK
```
Comment (dsbilling) [inline .env.example]: Key should NOT be stored in example!

File: app/Helpers/XenforoHelper.php [hunk 4]
```diff
@@ -21,22 +20,29 @@ private static function sendRequest($endpoint, $data = [], $method = self::METHO
             $method = $data;
             $data = [];
         }
-        $url = self::BASE_URL . $endpoint;
+        $url =  config('xenforo.base_url') . $endpoint;
```
Comment (dsbilling) [inline app/Helpers/XenforoHelper.php]: Double space?

File: resources/views/partials/newsbox.blade.php [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
-    <h2 class="col-12  text-center">Latest news</h2>
-    @foreach(\App\CustHelpers\XenforoHelper::getNewsItems()['threads'] as $item)
+    <h2 class="col-12 text-dark text-center crystal-textbox">Latest news</h2>
+    @forelse(\App\CustHelpers\XenforoHelper::getNewsItems()['threads'] as $item)
         @if($loop->iteration > 5)
             @break
-            @endif
-            <a href="https://pokemon3d.net/forum/threads/{{$item['thread_id']}}" class="col-12 pb-2 showoff_item">
+        @endif
+            <a href="https://pokemon3d.net/forum/threads/{{$item['thread_id']}}" class="col-12 crystal-textbox pb-2 showoff_item">
```
Comment (dsbilling) [inline resources/views/partials/newsbox.blade.php:7]: Maybe get this URL from env/config too? At least the base url for the forum

File: resources/views/partials/statistics.blade.php [hunk 6]
```diff
@@ -1,21 +1,21 @@
 <div class="col-12 row p-3 ">
-    <h2 class="col-12  text-center">Statistics</h2>
+    <h2 class="col-12 text-dark  crystal-textbox text-center">Statistics</h2>
 
-    <a href="#FORUM_LINK" class="col-12 showoff_item pb-2 pt-2">
+    <a href="https://pokemon3d.net/forum/" class="col-12 showoff_item crystal-textbox pb-2 pt-2">
```
Comment (dsbilling) [inline resources/views/partials/statistics.blade.php]: Maybe get this URL from env/config too? At least the base url for the forum

**Review by dsbilling — CHANGES_REQUESTED** [review]

**Pull request merged**